automated/utils/test-runner.py

```diff
@@ -5,6 +5,7 @@
 import copy
 import json
 import logging
+import netrc
```

Comment:
netrc has a strange effect on python requests. I need to double check if this is fixed in squad_client (should be).

Reply:
Seconded - please don't use netrc. Even if squad_client fixes for it, any other things that use requests may become broken. Anyway, shouldn't squad_client handle the token gathering?

Reply:
Yes the token from netrc causes issue while uploading results to qa-reports. It returns a sign-in page instead. I have dropped reading token from .netrc from the patch

Reply:
It's a 'feature' in requests. You can work around it this way:
https://github.com/psf/requests/issues/2773#issuecomment-174312831

Reply:
Ok I tried enabling the .netrc back. The token from .netrc was working last week but now I get this:
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Linaro QA - </title>

    <link rel="shortcut icon" href="/static/favicon.216d7ce1b499.ico"/>

    <link rel="stylesheet" href="/static/bootstrap/css/bootstrap.e7aa4e830a64.css">
    <link rel="stylesheet" href="/static/bootstrap/css/bootstrap-theme.b9b46bcc4dad.css">
    <link rel="stylesheet" href="/static/font-awesome/css/font-awesome.7d46203b9170.css">
    <link rel="stylesheet" href="/static/main.3dca4f718002.css">
    

  </head>

  <body>

    <div id='wrapper'>
    <nav id='navbar' class="navbar navbar-default">
      <div class="container container-fluid">
        <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
        <a class="btn btn-navbar navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </a>
        <div class="navbar-header">
          <a class="navbar-brand" href="/">
            <span class="fa fa-line-chart" aria-hidden="true"></span>
            Linaro QA
          </a>
        </div>

        <ul class="nav navbar-nav collapse navbar-collapse">
          <li class="None"><a href="/">Groups</a></li>
          <li class="dropdown">
            <a class="dropdown-toggle" href="#" id="compareDropdown" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              Compare
              <span class="caret"></span>
            </a>
            <ul id="compare-menu" class="dropdown-menu" aria-labelledby="compareDropdown">
              <li>
                  <a class="dropdown-item" href="/_/comparebuilds/">Compare Builds</a>
              </li>
              <li role="separator" class="divider"></li>
              <li>
                  <a class="dropdown-item" href="/_/compare/">Compare Projects</a>
              </li>
              <li role="separator" class="divider"></li>
              <li>
                  <a class="dropdown-item "href="/_/comparetest/">Compare Test</a>
              </li>
            </ul>
          </li>
          <li class="None"><a href="/api/">API</a></li>
        </ul>

        <ul class="nav navbar-nav collapse navbar-collapse navbar-right">
          


  <li>
    <a href="/login/?next=/api/submit/staging-lkft/aosp-master-vts-tracking/testbuild2/hi6220-hikey - arm64 (AOSP master)"><i class='fa fa-sign-in'></i> Log in</a>
  </li>

        </ul>

      </div>
    </nav>

    <div id="main-content">
      <div class="container">
        

<h1>Authentication required</h1>
<p>
<a href="/login?next=/api/submit/staging-lkft/aosp-master-vts-tracking/testbuild2/hi6220-hikey - arm64 (AOSP master)">Go to the login page.</a>
</p>



      </div>
    </div>

    <footer id='footer'>
      <div class="container">
        <a href="http://squad.readthedocs.io/">SQUAD</a> version 1.11.
        SQUAD is free software, distributed under the terms of <a href="https://www.gnu.org/licenses/agpl-3.0.html">GNU Affero General Public License, version 3</a> or (at your option) any later version.
        <a href="https://github.com/Linaro/squad">Source code</a> and <a href="https://github.com/Linaro/squad/issues">Issue tracker</a> are available on Github.
      </div>
    </footer>
    </div><!-- id="wrapper" -->

    <script type="text/javascript" src="/static/jquery.09dd64a64ba8.js"></script>
    <script type="text/javascript" src="/static/bootstrap/js/bootstrap.fb81549ee289.js"></script>
    <script type="text/javascript" src="/static/angularjs/angular.79054c75a5ca.js"></script>
    <script type="text/javascript" src="/static/lodash.a361a74fb3b7.js"></script>
    <script type="text/javascript" src="/static/floatThead/dist/jquery.floatThead.0bd7f184d926.js"></script>
    <script type="text/javascript" src="/static/squad/common.e41afb004503.js"></script>
    <script type="text/javascript">
        csrf_token = 'bCOXts0t7dNTIywh2lEdznuII0wJQmIqnUdHEhDV7Qk2CkHPuyDmdnbFu9WMJLPp'
        user = 'AnonymousUser'
    </script>
    
  </body>
</html>